Add Mojo Documentation (#90)
mulesoft/docs-mule-sdk: modules/ROOT/pages/rest-sdk/project-setup.adoc

include::../_attributes.adoc[]
endif::[]

To create a connector using Rest SDK:

> * Use a Mojo to scaffold a Connector Descriptor for an API Spec.

Reviewer: ```suggestion
* Use a _mojo_ (Maven plain Old Java Object) to scaffold a Connector Descriptor for an API Spec.
```

> * Use a Mojo to create a Connector from an API Spec and a Connector Descriptor.

Reviewer: ```suggestion
* Use a mojo to create a Connector from an API Spec and a Connector Descriptor.
```

* Perform some modifications to the Connector Descriptor for tailoring the generated connector.
* Package the connector using Maven.
* Publish the connector to Anypoint Exchange.

== Prerequisites

* Java 8 SDK
* Apache Maven (3.3.9 or newer recommended)

== Install the Rest SDK Plugin

> A mojo is a **M**aven plain **O**ld **J**ava **O**bject.

Reviewer: ```suggestion
A _mojo_ is a Maven plain Old Java Object.
```

> Each mojo is an executable goal in Maven, and a *plugin* is a distribution of one or more related mojos.

Reviewer: ```suggestion
Each mojo is an executable goal in Maven, and a _plugin_ is a distribution of one or more related mojos.
```

> *This step is optional* as Maven will automatically download the plugin when you try to use it
> but it is useful as it allows you to be sure that you have access to the repositories and
> have the latest version.

Reviewer: ```suggestion
Maven automatically downloads the plugin when you try to use it. Optionally, you can run the following command to ensure you have access to necessary repositories and the latest plugin version:
```
